packages/api/src/command/medical/patient/convert-fhir-to-cda.ts

```diff
@@ -1,34 +1,35 @@
+import { Organization } from "@medplum/fhirtypes";
 import { out } from "@metriport/core/util/log";
 import { capture } from "@metriport/core/util/notifications";
 import { makeFhirToCdaConverter } from "../../../external/fhir-to-cda-converter/converter-factory";
-import { toFHIR as toFHIROrganization } from "../../../external/fhir/organization";
 import { Bundle } from "../../../routes/medical/schemas/fhir";
-import { getOrganizationOrFail } from "../organization/get-organization";
 
 export async function convertFhirToCda({
   cxId,
   patientId,
   docId,
   validatedBundle,
+  fhirOrganization,
+  orgOid,
```

Comment:
Just looking at this and stepping deeper into the path to `convertFhirBundleToCda`, the organization here isnt used in the converter, but rather only for uploading the  metadata file. 

I think we should move the logic to get the fhirOrganization to that uploading module if possible to keep the convertFhirToCda as "pure" as possible. 
I think it would even be better if the `uploadCdaDocuments` wasnt inside the implementations of`FhirToCdaConverter`  but that refactor might be a little too annoying - your choice.